pkg/kindsys/report.go

```diff
@@ -18,9 +20,26 @@ import (
 	"github.com/grafana/grafana/pkg/plugins/pfs/corelist"
 	"github.com/grafana/grafana/pkg/plugins/plugindef"
 	"github.com/grafana/grafana/pkg/registry/corekind"
+	"github.com/grafana/thema"
 )
 
-const reportFileName = "report.json"
+const (
+	// Program's output
+	reportFileName = "report.json"
+
+	// External references
+	repoBaseURL = "https://github.com/grafana/grafana/tree/main"
+	docsBaseURL = "https://grafana.com/docs/grafana/next/developers/kinds"
+
+	// Local references
+	coreTSPath  = "packages/grafana-schema/src/raw/%s/%s/%s_types.gen.ts"
+	coreGoPath  = "pkg/kinds/%s/%s_kind_gen.go"
```

Comment:
this is the path to the kind wrapper/thema bindings, not the raw generated types

That said, i wonder if linking to the directory is actually better? Linking to the directory implicitly sends the user the message "everything in this dir is potentially relevant"